support/routes/v2/isobot/ops/botService.ts

```diff
@@ -19,13 +27,84 @@ export interface SlackPayload {
   text: string
 }
 
+// Slack only gives us 3 seconds to respond, but usually DNS resolution is fast,
```

Comment:
uhm. why put an artificial limit for us here ah? is it for graceful error recovery?

Reply:
Clarified offline. This is because we have to handle the case where the nameserver itself is taking a very long time to respond (or potentially does not respond at all). The Slack command will timeout after 3 seconds and any additional time spent processing would have been wasted work, so opted to terminate early and get the user to retry the command.